site/src/comparison.rs

```diff
@@ -658,7 +658,30 @@ impl BenchmarkVariance {
         deltas
     }
 
-    fn upper_fence(&self) -> f64 {
+    // This is an absolute value indicating the noise barrier for changes on
+    // this benchmark.
+    //
+    // A number line could be divded like this:
+    //
+    // ------o-------o----------
+    //       ^   ^   ^
+    //       |   |   |
+    //       |   |   |
+    //       |   |   ---- +significance_threshold
+    //       |   |
+    //       |   - not significant, includes zero
+    //       |
+    //       ---- -significance_threshold()
+    fn significance_threshold(&self) -> f64 {
+        let (q1, median, q3) = self.quartiles();
+
+        // Changes that are IQR_MULTIPLIER away from the median are considered
```

Comment:
Can you talk more about why you changed calculation from `q3 + IQR` to `median + IQR`? I believe [outlier fences](https://www.statology.org/upper-and-lower-fences/) are normally calculated from the quartiles and not from the median.

Reply:
Hm. So my reasoning here was that it *felt* like using the median is somehow "better" -- adding IQR to the median seems more reasonable than adding it to Q3, which sort of already includes part of the IQR. This was also partially based on my (incorrect) recollection was that my statistics classes did have it based on the median, though since the pages you've linked to do all have q3, I was probably just wrong here :)

In practice our median and q3 are usually pretty close, so this probably has a negligible difference.

Reply:
Hmm I'm not sure if it feels better to me. Perhaps you could try to explain more why it does?

In any case if it's unlikely to make a huge difference, perhaps we should just stick with the text book definition?

Reply:
I'm happy to revert this particular change.

I think it mostly felt like we ended up with something like

`median + IQR * (some coefficient representing how "right skewed" we are) + IQR * IQR_MULTIPLIER`

where the middle term is the Q3-median delta. That felt a little weird for our purposes, where we don't actually care about the right-skew that much, in practice we will always have high right skew since all data is going to be bound to [0, infinity) so there's practically no way to be left skewed.

Reply:
That makes sense. I don't feel too strongly about this which I guess is why my gut says to fallback to the book mechanism, but I'll leave it to you. 

However, can you update the docs where we explain this? https://github.com/rust-lang/rustc-perf/blob/master/docs/comparison-analysis.md#what-makes-a-test-result-significant

Reply:
I'll just revert this change I think, that seems best. I'll update the constant 1.5 in the docs as well.

Reply:
https://github.com/rust-lang/rustc-perf/pull/1008